### PR TITLE
fix CCScale9Sprite for rotated sprite frame

### PR DESCRIPTION
**Pull request by smilingpoplar**
fix issue #1145

### CODE REVIEW

File: extensions/GUI/CCControlExtension/CCScale9Sprite.cpp [hunk 1]
```diff
@@ -138,6 +138,7 @@ bool Scale9Sprite::updateWithBatchNode(SpriteBatchNode* batchnode, Rect rect, bo
     _scale9Image->removeAllChildrenWithCleanup(true);
 
     _capInsets = capInsets;
+    m_bSpriteFrameRotated = rotated;
```
Comment (dumganhar) [inline extensions/GUI/CCControlExtension/CCScale9Sprite.cpp:141]: Where was `m_bSpriteFrameRotated` defined?

Comment (smilingpoplar) [inline extensions/GUI/CCControlExtension/CCScale9Sprite.cpp:141]: v2.1.4 has this `m_bSpriteFrameRotated` defined, https://github.com/cocos2d/cocos2d-x/blob/cocos2d-x-2.1.4/extensions/GUI/CCControlExtension/CCScale9Sprite.cpp

I didn't notice you guys have remove this var. Why remove it? 

Should I clone v2.1.4 and push to v2.1.4? I'm not clear about your git workflow. Anyway, adding this line to v2.1.4 works for me.

Comment (dumganhar) [inline extensions/GUI/CCControlExtension/CCScale9Sprite.cpp:141]: It was not removed, but renamed.

James Chen
ÔÚ 2013-7-3 ÏÂÎç11:46£¬"YangLe" notifications@github.comÐ´µÀ£º

Comment (smilingpoplar) [inline extensions/GUI/CCControlExtension/CCScale9Sprite.cpp:141]: alright, let me create a new pull

**Pull request closed without merging**